src/desktop/package.json

```diff
@@ -115,6 +115,11 @@
         }
       ]
     },
+    "nsis": {
+      "oneClick": false, 
+      "allowToChangeInstallationDirectory": true,
+      "deleteAppDataOnUninstall": true
```

Comment:
According to the [`electron-builder` docs](https://www.electron.build/configuration/nsis), `deleteAppDataOnUninstall` is only available when using the one-click installer

Reply:
:( I'll leave `deleteAppDataOnUninstall` only then as it's more required than changing the installation directory.